tasks/source-install.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Install required packages
```

Comment:
@akhon thoughts about baking this into the base image?

Reply:
why not?